EventVisualisation/Base/src/ConfigurationManager.cxx

```diff
@@ -0,0 +1,61 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See https://alice-o2.web.cern.ch/ for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    ConfigurationManager.cxx
+/// \author  Jeremi Niedziela
+///
+
+#include "EventVisualisationBase/ConfigurationManager.h"
+
+#include <TSystem.h>
+
+#include <iostream>
+
+using namespace std;
+
+namespace o2  {
+namespace EventVisualisation {
+  
+ConfigurationManager* ConfigurationManager::sInstance  = nullptr;
+
+ConfigurationManager* ConfigurationManager::getInstance()
+{
+  if(!sInstance){new ConfigurationManager();}
+  return sInstance;
+}
+
+ConfigurationManager::ConfigurationManager()
+{
+  sInstance = this;
+}
+  
+ConfigurationManager::~ConfigurationManager()
+{
+}
+
+void ConfigurationManager::getConfig(TEnv &settings)
```

Comment:
looks like a `const` function

Reply:
This is a static function - can't be const.

Reply:
True, didn't pay attention to the header file. But if this function is static and the only relevant code, the class and the singleton implementation is probably obsolete?

Reply:
Right, actually I prefer to keep it a singleton and change this function to regular one, because in the future I plan to expand this class and I would like to avoid ending up with too many static functions.